Tests/IT.New-ConfigurationTableEntry.Helpers.Tests.ps1

```diff
@@ -0,0 +1,60 @@
+Describe "New-ConfigurationTableEntry.Helper Integration Tests" -Tags @("Integration") {
+
+    BeforeAll {
+        Set-Location $PSScriptRoot/../Infrastructure-Scripts/New-ConfigurationTableEntry/
+        Import-Module ./tools/Helpers.psm1 -Force
+    }
+
+    Context "A valid schema and complete set of environment variables exists" {
+        if (!$ENV:TF_BUILD) {
+            Import-Module powershell-yaml
```

Comment:
Is a comment required to say that this module requires installing beforehand for visibility?

Reply:
done